ldi-core/ngsiv2-to-ld/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/config/NgsiV2ToLdMapping.java

```diff
@@ -0,0 +1,52 @@
+package be.vlaanderen.informatievlaanderen.ldes.ldi.config;
+
+import java.util.Map;
+
+public class NgsiV2ToLdMapping {
```

Comment:
Not visible in GH PRs, but I notice some of these fields are no longer used. Still a small cleanup needed perhaps?